python/capellaAPI/CapellaAPIAuth.py

```diff
@@ -10,7 +10,7 @@
 # Other Libs
 
 # Owned
-from capellaAPI.CapellaExceptions import *
+from .CapellaExceptions import *
```

Comment:
Must admit it's been a good while since I've written Python but... does this work? Leading `.` looks a bit odd to me is all.

Reply:
if you drop a __init__.py  in the folder where you have your modules that you’re going to be importing from, then . works and also folder.module does as well.